server/models/Team.js

```diff
@@ -42,13 +42,13 @@ Team.prototype.createFirstCollection = async function(userId) {
 };
 
 Team.prototype.addAdmin = async function(user: User) {
-  return await user.update({ isAdmin: true });
+  return user.update({ isAdmin: true });
 };
 
 Team.prototype.removeAdmin = async function(user: User) {
   const res = await User.findAndCountAll({
     where: {
-      teamId: user.teamId,
+      teamId: this.id,
```

Comment:
The controller was checking the authenticated users team, but then the model was using the promoted users team – these are the same right now, but makes sense to make these match.